emgapiv2/test_data_privacy.py

```diff
@@ -0,0 +1,140 @@
+    public_study = Study.objects.create(title="Public Study", is_private=False)
+    private_study = Study.objects.create(title="Private Study", is_private=True)
+
+    url = reverse("admin:analyses_study_changelist")
+    response = admin_client.get(url)
+
+    assert response.status_code == 200
+    content = response.content.decode("utf-8")
+    assert public_study.title in content
+    assert private_study.title in content
+
+
+@pytest.mark.django_db(transaction=True)
+def test_admin_view_analyses(raw_read_run, admin_client):
+    """Test that admin interface shows all analyses"""
+    public_analysis = create_analysis(is_private=False)
+    private_analysis = create_analysis(is_private=True)
+
+    url = reverse("admin:analyses_analysis_changelist")
+    response = admin_client.get(url)
+
+    assert response.status_code == 200
+    content = response.content.decode("utf-8")
+    assert public_analysis.accession in content
+    assert private_analysis.accession in content
+
+
+@pytest.mark.django_db(transaction=True)
+def test_study_manager_methods():
+    # def test_manager_methods(raw_read_run):
+    """Test various manager methods for privacy handling"""
+    ena_study = ENAStudy.objects.create(accession="PRJ1", title="Project 1")
+    public_study = Study.objects.create(
+        ena_study=ena_study, title="Public Study", is_private=False
+    )
+    private_study = Study.objects.create(title="Private Study", is_private=True)
+
+    assert Study.objects.count() == 1
+    assert Study.objects.first() == public_study
+    assert Study.all_objects.count() == 2
+    private_studies = Study.objects.private_only()
+    assert private_studies.count() == 1
+    assert private_studies.first() == private_study
+
+
+@pytest.mark.django_db(transaction=True)
+def test_manager_analysis_methods(raw_read_run):
+    public_analysis = create_analysis(is_private=False)
+    private_analysis = create_analysis(is_private=True)
+
+    assert Analysis.objects.count() == 1
+    assert Analysis.objects.first() == public_analysis
+    assert Analysis.all_objects.count() == 2
+    private_analyses = Analysis.objects.private_only()
+    assert private_analyses.count() == 1
+    assert private_analyses.first() == private_analysis
+
+    public_analysis.annotations = {"test": "data"}
+    public_analysis.save()
+
+    analysis = Analysis.objects_and_annotations.first()
+    assert analysis.annotations == {"test": "data"}
+    all_analyses = Analysis.all_objects_and_annotations.all()
+    assert len(all_analyses) == 2
+
+
+@pytest.fixture
+def api_client():
+    from rest_framework.test import APIClient
+
+    return APIClient()
+
+
+@pytest.fixture
+def admin_client():
+    from django.test import Client
+
+    admin_user = User.objects.create_superuser(
+        username="admin", email="admin@example.com", password="adminpass123"
+    )
+    client = Client()
+    client.force_login(admin_user)
+    return client
```

Comment:
There should already be an `api_client` and `admin_client` fixture provided by `pytest-django` automatically - we shouldn't need to define these ones.

Reply:
found admin-client, and thus removed my implementation.
However, I  could not find an api-client fixture out of the box. So I've maintained that